src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -67,7 +71,7 @@ case class Profile (
 object Profile {
 
   // increment this number every time you make a change to the user-provided profile fields
-  val currentVersion:Int = 2
+  val currentVersion:Int = 3
```

Comment:
i don't have a better solution and i'm not requesting you change it, but i'd like to register my extreme displeasure at this. it's horrifying